css/why-dine.css

```diff
@@ -184,7 +184,41 @@
   display: block;
 }
 
-/* Customers Favourite Section */
+
+.playbtn {
+  font-size: 24px;
+  color: white;
+  background-color: rgba(232, 204, 122, 0.9);
+  padding: 20px;
+  border-radius: 50%;
+  border: none;
+  cursor: pointer;
+  width: 80px;
+  height: 80px;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  transition: all 0.3s ease;
+  box-shadow: 0 4px 15px rgba(0, 0, 0, 0.3);
+  backdrop-filter: blur(10px);
+}
+
+.playbtn:hover {
+  background-color: #f1d88c;
+  color: #222;
+  transform: scale(1.1);
+  box-shadow: 0 6px 20px rgba(241, 216, 140, 0.5);
+}
+
+.playbtn-div{
```

Comment:
`.playbtn` is genoeg. Als je een wrapper er omheen bedoelt, maak er dan van `.playbtn-wrapper` dat zegt wat meer.